.circleci/config.yml

```diff
@@ -1099,6 +1109,17 @@ workflows:
               audit_deps
             ]
           filters: *master_and_version_branches_and_all_tags
+
+      - publish_perf:
```

Comment:
Probably like publish_watcher, only publish after some required tests have been done? eg the local docker perf test one so we can make sure this version of perf is runnable.

Reply:
I added  `requires: [test_docker_compose_performance]`